src/Runtime/OMExternalConstant.inc

```diff
@@ -89,7 +93,10 @@ void omMMapBinaryFile(
       filePath = (char *)malloc(filePathLen);
```

Comment:
You probably want to +1 to `filePathLen` to account for the `\0` you add to the end later.

Reply:
Good catch! I updated this.